### PR TITLE
Describes the Authority node process

### PR DESCRIPTION
**Pull request by betarelease**
Answers questions that came up during the retro - https://github.com/pyrsia/pyrsia/issues/1337

### CODE REVIEW

File: blog/2023-01-05-authority_nodes.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Authority nodes act as the only legal votes in the Proof of Authority to achieve Consensus on Pyrsia blockchain. Authority nodes are assigned keys to prove membership and they vote using those keys to verify packages on the Pyrsia network. These assigned keys are controlled and regenerated by the Pyrsia governance board. We explain here the process and the technology required to implement this.
+
+## Pyrsia Security Board
+
+Pyrsia project is working towards this concept of having a group of representatives forming a Security Board to ensure the security of the network and the artifacts served by the network. This board will have the scope of providing process and oversight over the following:
+
+1. Pyrsia Network Architecture
+2. Pyrsia Network Security
+3. Pyrsia Blockchain Xecurity
```
Comment (dasmanas) [inline blog/2023-01-05-authority_nodes.md:17]:
```suggestion
3. Pyrsia Blockchain Security
```

File: blog/2023-01-05-authority_nodes.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+4. Pyrsia Devops Practices
+5. Pyrsia Build infrastructure Security
```
Comment (dasmanas) [inline blog/2023-01-05-authority_nodes.md:19]:
```suggestion
5. Pyrsia Build Infrastructure Security
```

File: blog/2023-01-05-authority_nodes.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+tags: [authority, consensus, proof of authority, cloud, deployment, immutable ledger, BFT, security board]
+---
+
+Pyrsia builds consensus based on Proof of Authority which is exercised by the Authority Nodes. Authority nodes are run by trusted Partner organizations who join the Pyrsia community and decide to contribute to the network. More about this architecture was detailed in a the [Network Structure of Pyrsia](./2022-10-10-why-does-pyrsia-use-a-blockchain/#network-structure>).
+
+Authority nodes act as the only legal votes in the Proof of Authority to achieve Consensus on Pyrsia blockchain. Authority nodes are assigned keys to prove membership and they vote using those keys to verify packages on the Pyrsia network. These assigned keys are controlled and regenerated by the Pyrsia governance board. We explain here the process and the technology required to implement this.
+
+## Pyrsia Security Board
+
+Pyrsia project is working towards this concept of having a group of representatives forming a Security Board to ensure the security of the network and the artifacts served by the network. This board will have the scope of providing process and oversight over the following:
```
Comment (dasmanas) [inline blog/2023-01-05-authority_nodes.md:13]:
```suggestion
Pyrsia project is working towards the concept of having a group of representatives forming a Security Board to ensure the security of the network and the artifacts served by the network. This board will have the scope of providing process and oversight over the following but not limited to:
```

File: blog/2023-01-05-authority_nodes.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+---
+title: Authority Nodes
+authors: sudhindra
+tags: [authority, consensus, proof of authority, cloud, deployment, immutable ledger, BFT, security board]
+---
+
+Pyrsia builds consensus based on Proof of Authority which is exercised by the Authority Nodes. Authority nodes are run by trusted Partner organizations who join the Pyrsia community and decide to contribute to the network. More about this architecture was detailed in a the [Network Structure of Pyrsia](./2022-10-10-why-does-pyrsia-use-a-blockchain/#network-structure>).
+
+Authority nodes act as the only legal votes in the Proof of Authority to achieve Consensus on Pyrsia blockchain. Authority nodes are assigned keys to prove membership and they vote using those keys to verify packages on the Pyrsia network. These assigned keys are controlled and regenerated by the Pyrsia governance board. We explain here the process and the technology required to implement this.
```
Comment (dasmanas) [inline blog/2023-01-05-authority_nodes.md:9]:
```suggestion
Authority nodes act as the only legal votes in the Proof of Authority to achieve Consensus on Pyrsia blockchain. Authority nodes are assigned keys to prove membership and they vote using those keys to verify packages in the Pyrsia network. These assigned keys are controlled and regenerated by the Pyrsia governance board. We explain here the process and the technology required to implement this.
```

File: blog/2023-01-05-authority_nodes.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+Pyrsia project is working towards this concept of having a group of representatives forming a Security Board to ensure the security of the network and the artifacts served by the network. This board will have the scope of providing process and oversight over the following:
+
+1. Pyrsia Network Architecture
+2. Pyrsia Network Security
+3. Pyrsia Blockchain Xecurity
+4. Pyrsia Devops Practices
+5. Pyrsia Build infrastructure Security
+
+Within the above scope the Security Board will provide a uniform review and regulation process to ensure that rogue elements are not able to control the network. This Security board will be formed with representative organizations to Pyrsia via an election process.
```
Comment (dasmanas) [inline blog/2023-01-05-authority_nodes.md:21]:
```suggestion
Within the above scope the Security Board will provide a uniform review and regulation processes to ensure that rogue elements are not able to control the network. This Security board will be formed with the representative organizations to Pyrsia via an election process.
```

**Review by dasmanas — APPROVED** [review]

**Review by fishseabowl — APPROVED** [review]

File: blog/2023-01-05-authority_nodes.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+---
+title: Authority Nodes
+authors: sudhindra
+tags: [authority, consensus, proof of authority, cloud, deployment, immutable ledger, BFT, security board]
+---
+
+Pyrsia builds consensus based on Proof of Authority which is exercised by the Authority Nodes. Authority nodes are run by trusted Partner organizations who join the Pyrsia community and decide to contribute to the network. More about this architecture was detailed in a the [Network Structure of Pyrsia](./2022-10-10-why-does-pyrsia-use-a-blockchain/#network-structure>).
```
[the remark this answers is not in the document]
Comment (betarelease) [reply blog/2023-01-05-authority_nodes.md]: fixed

File: blog/2023-01-05-authority_nodes.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (betarelease) [reply blog/2023-01-05-authority_nodes.md]: corrected

File: blog/2023-01-05-authority_nodes.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (betarelease) [reply blog/2023-01-05-authority_nodes.md]: corrected

File: blog/2023-01-05-authority_nodes.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+As part of the normal functioning of this board, the Pyrsia infrastructure will be evaluated periodically and certified.
+
+Details of this process will be documented separtely under the Community section of the [Pyrsia website](https://pyrsia.io).
+
+## Participating Authority Nodes
+
+Any participating organization to Pyrsia is welcome to join the network and host/sponsor hosting an Authority node. Offering to sponsor an authority node will come with the following responsibilities:
+
+1. Sponsor the cloud account and billing to host a large size Pyrsia Authority node - specifications will be provided and updated by the Security Board
+2. Sponsor and provide Devops personnel to maintain, update and repair the Pyrsia Authority node
```
[the remark this answers is not in the document]
Comment (betarelease) [reply blog/2023-01-05-authority_nodes.md]: corrected

File: blog/2023-01-05-authority_nodes.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+3. Submit audit reports to the Security Board to ensure that the Authority node remains secure
+
+Since Pyrsia Authority Node is open source the source code, helm charts, deployment processes will be maintained by the Pyrsia team thus making it  easy to install and run the Pyrsia Authority Node in the cloud environment of choice.
```
[the remark this answers is not in the document]
Comment (betarelease) [reply blog/2023-01-05-authority_nodes.md]: fixed with punctuation to make it clear.

File: blog/2023-01-05-authority_nodes.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+## Participate in Proof of Authority
+
+Once the infrastructure is up an running on the cloud the Pyrsia nodes become part of the Pyrsia libp2p network and can download, search the Pyrsia network. In order to make these Pyrsia nodes part of the Consensus network this deployment will require some way to gain that authority.
+
+Since we need to maintain a small but reliable and tamperproof network access to authority needs to and will be limited. A people driven process will be in place that determines the readiness of this new participating organization. The security board will drive this process and lay out guidelines to make it transparent.
+The security board will also have the ability to assign authority keys to new participant organizations so that they can become part of the consensus process.
+
+Each organization will receive one key that will allow them to participate in voting and pulishing packages to the network. This publishing process will allow them to commit new versions of packages, new packages to the Pyrsia network.
```
[the remark this answers is not in the document]
Comment (betarelease) [reply blog/2023-01-05-authority_nodes.md]: fixed

File: blog/2023-01-05-authority_nodes.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+1. Sponsor the cloud account and billing to host a large size Pyrsia Authority node - specifications will be provided and updated by the Security Board
+2. Sponsor and provide Devops personnel to maintain, update and repair the Pyrsia Authority node
+3. Submit audit reports to the Security Board to ensure that the Authority node remains secure
+
+Since Pyrsia Authority Node is open source the source code, helm charts, deployment processes will be maintained by the Pyrsia team thus making it  easy to install and run the Pyrsia Authority Node in the cloud environment of choice.
+
+## Participate in Proof of Authority
+
+Once the infrastructure is up an running on the cloud the Pyrsia nodes become part of the Pyrsia libp2p network and can download, search the Pyrsia network. In order to make these Pyrsia nodes part of the Consensus network this deployment will require some way to gain that authority.
```
[the remark this answers is not in the document]
Comment (betarelease) [reply blog/2023-01-05-authority_nodes.md]: fixed

[the remark this answers is not in the document]
Comment (betarelease) [reply blog/2023-01-05-authority_nodes.md]: artifacts are not the only thing Pyrsia has hence did not accept that change.

File: blog/2023-01-05-authority_nodes.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+The security board will also have the ability to assign authority keys to new participant organizations so that they can become part of the consensus process.
+
+Each organization will receive one key that will allow them to participate in voting and pulishing packages to the network. This publishing process will allow them to commit new versions of packages, new packages to the Pyrsia network.
+
+## Growing the network
+
+Pyrsia released the first working version of the network in our [0.2.0 release](https://github.com/pyrsia/pyrsia/releases/tag/v0.2.0) which included an Authority node hosted by JFrog. We are working towards adding another Authority node that will run on a different cloud provider's infrastructure and will be releasing it soon.
+
+In the meanwhile we are also encouraging organizations to become part of the Pyrsia network and contribute Authoriy nodes and infrastructure to the network. If you are one of those organizations we would love to chat with you. Our goal is to make this network as distributed as possible - which allows Pyrsia artifacts to reach all the corners of the known internet as well as helps to keep the network running.
```
[the remark this answers is not in the document]
Comment (betarelease) [reply blog/2023-01-05-authority_nodes.md]: fixed in a different way.

File: blog/2023-01-05-authority_nodes.md (hunk 3 above)
Comment (betarelease) [reply blog/2023-01-05-authority_nodes.md]: Dont want to add legal statements here. So skipping the `but not limited to` suggestion.

File: blog/2023-01-05-authority_nodes.md (hunk 5 above)
Comment (betarelease) [reply blog/2023-01-05-authority_nodes.md]: fixed.

File: blog/2023-01-05-authority_nodes.md [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+2. Sponsor and provide Devops personnel to maintain, update and repair the Pyrsia Authority node
+3. Submit audit reports to the Security Board to ensure that the Authority node remains secure
+
+Since Pyrsia Authority Node is open source the source code, helm charts, deployment processes will be maintained by the Pyrsia team thus making it  easy to install and run the Pyrsia Authority Node in the cloud environment of choice.
+
+## Participate in Proof of Authority
+
+Once the infrastructure is up an running on the cloud the Pyrsia nodes become part of the Pyrsia libp2p network and can download, search the Pyrsia network. In order to make these Pyrsia nodes part of the Consensus network this deployment will require some way to gain that authority.
+
+Since we need to maintain a small but reliable and tamperproof network access to authority needs to and will be limited. A people driven process will be in place that determines the readiness of this new participating organization. The security board will drive this process and lay out guidelines to make it transparent.
```
[the remark this answers is not in the document]
Comment (betarelease) [reply blog/2023-01-05-authority_nodes.md]: fixed. removed the confusing and.

File: blog/2023-01-05-authority_nodes.md (hunk 1 above)
Comment (betarelease) [reply blog/2023-01-05-authority_nodes.md]: fixed

**Pull request merged**